src/daemon/rpmostreed-os.c

```diff
@@ -104,9 +204,13 @@ static void
 os_constructed (GObject *object)
 {
   RpmostreedOS *self = RPMOSTREED_OS (object);
+  GError *error = NULL;
 
-  /* TODO Integrate with PolicyKit via the "g-authorize-method" signal. */
-
+  self->authority = polkit_authority_get_sync (NULL, &error);
+  if (self->authority == NULL)
+    {
+      g_error ("Can't get polkit authority: %s", error->message);
```

Comment:
I tend to avoid `g_error()` in general because few conditions where one typically finds it necessitate dumping core.  That's true here; if we fail to talk to polkitd, it's highly unlikely that a core dump from rpm-ostreed is going to be useful to operators.

Hence I'd prefer `errx()` in this case.

(And in cases where we *do* want a core dump, that's `g_assert()`)